packages/bento-design-system/src/util/atoms.ts

```diff
@@ -89,6 +89,7 @@ const color = {
   ...vars.interactiveForegroundColor,
   ...vars.outlineColor,
   ...vars.dataVisualizationColor,
+  transparent: "transparent",
```

Comment:
we had to add `transparent` to `color` to be able to pass such value to the navigation's `Box`

Reply:
@gabro in order to use the a foreground color as a background, we're using the `background="currentColor"` trick. This means the decorator color is passed to the `color` Box prop, but decorator colors support the additional "transparent" value that is not supported by `color`. Thus, we added it to the available colors too, meaning that now it's theoretically possible to use "transparent" as a foreground color. Anything against it?